{
  "api": "search",
  "file": "/test/free/data_stream/210_rollover_failure_store.yml",
  "name": "Lazily roll over a data stream's failure store after a shard failure",
  "origin": "yaml",
  "request": {
    "args": {
      "index": ".fs-data-stream-for-rollover-*",
      "typed_keys": true
    }
  },
  "response": {
    "headers": {
      "content-type": "application/json",
      "transfer-encoding": "chunked",
      "x-elastic-product": "Elasticsearch"
    },
    "payload": {
      "_shards": {
        "failed": 0,
        "skipped": 0,
        "successful": 2,
        "total": 2
      },
      "hits": {
        "hits": [
          {
            "_id": "ZVUrV5kBl9s8-GIpseJB",
            "_index": ".fs-data-stream-for-rollover-2025.09.17-000002",
            "_score": 1,
            "_source": {
              "@timestamp": "2025-09-17T10:15:05.088Z",
              "document": {
                "id": "ZFUrV5kBl9s8-GIpsOL1",
                "index": "data-stream-for-rollover",
                "source": {
                  "@timestamp": "2020-12-12",
                  "count": "invalid value"
                }
              },
              "error": {
                "message": "[1:36] failed to parse field [count] of type [long] in document with id 'ZFUrV5kBl9s8-GIpsOL1'. Preview of field's value: 'invalid value'",
                "stack_trace": "o.e.i.m.DocumentParsingException: [1:36] failed to parse field [count] of type [long] in document with id 'ZFUrV5kBl9s8-GIpsOL1'. Preview of field's value: 'invalid value'\n\tat o.e.i.m.FieldMapper.rethrowAsDocumentParsingException(FieldMapper.java:242)\n\tat o.e.i.m.FieldMapper.parse(FieldMapper.java:195)\n\t... 24 more\nCaused by: j.l.IllegalArgumentException: For input string: \"invalid value\"\n\tat o.e.x.s.AbstractXContentParser.toLong(AbstractXContentParser.java:201)\n\tat o.e.x.s.AbstractXContentParser.longValue(AbstractXContentParser.java:222)\n\t... 30 more\n",
                "type": "document_parsing_exception"
              }
            }
          },
          {
            "_id": "Z1UrV5kBl9s8-GIpseK2",
            "_index": ".fs-data-stream-for-rollover-2025.09.17-000003",
            "_score": 1,
            "_source": {
              "@timestamp": "2025-09-17T10:15:05.288Z",
              "document": {
                "id": "ZlUrV5kBl9s8-GIpseJr",
                "index": "data-stream-for-rollover",
                "source": {
                  "@timestamp": "2020-12-12",
                  "count": "invalid value"
                }
              },
              "error": {
                "message": "[1:36] failed to parse field [count] of type [long] in document with id 'ZlUrV5kBl9s8-GIpseJr'. Preview of field's value: 'invalid value'",
                "stack_trace": "o.e.i.m.DocumentParsingException: [1:36] failed to parse field [count] of type [long] in document with id 'ZlUrV5kBl9s8-GIpseJr'. Preview of field's value: 'invalid value'\n\tat o.e.i.m.FieldMapper.rethrowAsDocumentParsingException(FieldMapper.java:242)\n\tat o.e.i.m.FieldMapper.parse(FieldMapper.java:195)\n\t... 24 more\nCaused by: j.l.IllegalArgumentException: For input string: \"invalid value\"\n\tat o.e.x.s.AbstractXContentParser.toLong(AbstractXContentParser.java:201)\n\tat o.e.x.s.AbstractXContentParser.longValue(AbstractXContentParser.java:222)\n\t... 30 more\n",
                "type": "document_parsing_exception"
              }
            }
          }
        ],
        "max_score": 1,
        "total": {
          "relation": "eq",
          "value": 2
        }
      },
      "timed_out": false,
      "took": 2
    },
    "statusCode": 200
  }
}